examples/vitejs-with-typescript/README.md

```diff
@@ -0,0 +1,39 @@
+# Vite.js with Typescript example
+
+## How to use
+
+Download the example [or clone the repo](https://github.com/mui/material-ui):
+
+<!-- #default-branch-switch -->
+
+```sh
+curl https://codeload.github.com/mui/material-ui/tar.gz/next | tar -xz --strip=2 material-ui-next/examples/vitejs-with-typescript
```

Comment:
It can be downloaded from `master` branch.
```suggestion
curl https://codeload.github.com/mui/material-ui/tar.gz/master | tar -xz --strip=2 material-ui-master/examples/vitejs-with-typescript
```